core/chains/evm/config/v2/defaults/Optimism_Goerli.toml

```
@@ -21,3 +21,7 @@ BlockHistorySize = 0
 [HeadTracker]
 HistoryDepth = 10
 SamplingInterval = '1s'
+
+[OCR2]
```

I think it's probably cleaner to omit the extra table headers.
```suggestion
-[OCR2]
```

---

Oh, but our toml package still outputs them when a sub-table has fields... :thinking: 
I bet we could hack around that, but maybe just have to live with it.

---

yeah tests are failing. I'll revert this for now

---

Oh these ones should be fine, since they are only used for _input_, right? I think it is actually important to keep them as clean as possible. The test files that validate the output will have to match exactly though.